clients/drcachesim/tests/memref_gen.h

```diff
@@ -75,6 +75,15 @@ gen_branch(memref_tid_t tid, addr_t pc)
 // We use these client defines which are the target and so drdecode's target arch.
 #if defined(ARM_64) || defined(ARM_32)
 // Variant for aarchxx encodings.
+inline memref_t
+gen_instr_encoded(memref_tid_t tid, addr_t pc, int encoding, size_t size = 1)
+{
```

Comment:
About the `size` arg: AArch64 has fixed sized instrs.

Reply:
Ack. Removed the size arg and used 4 in the call to `gen_instr_type`